components/PageLayout/Home.vue

```diff
@@ -1,18 +1,15 @@
 <template>
   <div data-testid="homePageLayout">
-    <SectionSlider
-      v-if="page.data && page.data['pt_slider']"
-      :data="page.data['pt_slider']"
-    />
-    <PageBody
-      v-for="pageBody in page.children"
-      :key="pageBody.previewId"
-      :page-body="pageBody"
-    />
+    <PageBody :page-body="top" />
+    <PageBody :page-body="content" />
   </div>
 </template>
 
 <script setup lang="ts">
 import { Page } from 'fsxa-api'
-defineProps<{ page: Page }>()
+const props = defineProps<{ page: Page }>()
+const top = props.page.children.find((pagebody) => pagebody.name === 'top')!
```

Comment:
This is automatically generated by prettier and it depends on the number of characters in a line.
Basically there is an upper limit so the code fits on a certain screen size if a line is longer than the limit it will get split up if possible. The line below is longer than this one. I don't make the rules 🤷🏻‍♂️